doc/source/whatsnew/v0.25.0.rst

```diff
@@ -403,6 +403,7 @@ Groupby/Resample/Rolling
 - Bug in :meth:`pandas.core.groupby.GroupBy.idxmax` and :meth:`pandas.core.groupby.GroupBy.idxmin` with datetime column would return incorrect dtype (:issue:`25444`, :issue:`15306`)
 - Bug in :meth:`pandas.core.groupby.GroupBy.cumsum`, :meth:`pandas.core.groupby.GroupBy.cumprod`, :meth:`pandas.core.groupby.GroupBy.cummin` and :meth:`pandas.core.groupby.GroupBy.cummax` with categorical column having absent categories, would return incorrect result or segfault (:issue:`16771`)
 - Bug in :meth:`pandas.core.groupby.GroupBy.nth` where NA values in the grouping would return incorrect results (:issue:`26011`)
+- Bug in :meth:`pandas.core.groupby.BaseGrouper._cython_operation` where incorrect results are returned for uint64 columns. (:issue:`26310`)
```

Comment:
Whatsnew messages should be user-facing, i.e. only references items exposed via the API. This isn't, but I think you can refactor to reference `pandas.core.groupby.GroupBy.agg` instead

Reply:
fixed, thanks!